modules/k6/examples_test.go

```diff
@@ -0,0 +1,76 @@
+package k6_test
+
+import (
+	"context"
+	"fmt"
+	"path/filepath"
+
+	"github.com/testcontainers/testcontainers-go"
+	"github.com/testcontainers/testcontainers-go/modules/k6"
+	"github.com/testcontainers/testcontainers-go/wait"
+)
+
+func ExampleRunContainer() {
+	// runK6Container {
+	ctx := context.Background()
+
+	// create a container with the httpbin application that will be the target
+	// for the test script that runs in the k6 container
+	gcr := testcontainers.GenericContainerRequest{
+		ProviderType: testcontainers.ProviderDocker,
+		ContainerRequest: testcontainers.ContainerRequest{
+			Image: "kennethreitz/httpbin",
+			ExposedPorts: []string{
+				"80",
+			},
+			WaitingFor: wait.ForExposedPort(),
+		},
+		Started: true,
+	}
+	httpbin, err := testcontainers.GenericContainer(ctx, gcr)
+	if err != nil {
+		panic(fmt.Errorf("failed to create httpbin container %w", err))
+	}
+
+	defer func() {
+		if err := httpbin.Terminate(ctx); err != nil {
+			panic(fmt.Errorf("failed to terminate container: %w", err))
+		}
+	}()
+
+	httpbinIP, err := httpbin.ContainerIP(ctx)
+	if err != nil {
+		panic(fmt.Errorf("failed to get httpbin IP: %w", err))
+	}
+
+	absPath, err := filepath.Abs(filepath.Join("scripts", "httpbin.js"))
+	if err != nil {
+		panic(fmt.Errorf("failed to get path to test script: %w", err))
+	}
+
+	// run the httpbin.js test scripts passing the IP address the httpbin container
+	k6, err := k6.RunContainer(
+		ctx,
+		k6.WithTestScript(absPath),
```

Comment:
Let's include the cache here too, for others to see it in action since the first time:
```suggestion
		k6.WithTestScript(absPath),
		k6.WithCache("k6-cache"),
```

Reply:
I don't see the value here, as it runs only one test.

Reply:
Indeed, but the docs page serve as reference and many people go there to copy&paste code

Reply:
I agree, but there's also the documentation and I don't think all features can be demonstrated in one example without either making it too complex or abusing the options, like in this case the `WithCache` that would be of no benefit at all.

Another option would be to offer more than one example.

Reply:
Agreed. Whatever you prefer is fine then 👍